src/components/ItaliaTheme/Icons/FontAwesomeIcon.jsx

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 import { fontAwesomeAliases } from 'design-comuni-plone-theme/helpers/index';
 
 const FontAwesomeIcon = (props) => {
-  const { className, icon, prefix, title } = props;
+  const { className, icon, prefix, title, ariaHidden } = props;
```

Comment:
Consiglio pratico anche in ottica di ragionare con typescript e scrivere meno codice: l'attributo aria-hidden è un attributo comune tra gli elementi e i tag html in generale. React rispetta in questo caso (so che in molti altri vale la regola del camel case, ma qui no https://legacy.reactjs.org/docs/accessibility.html#wai-aria) il nome originale degli attributi aria-*. In ts, quelle props sarebbero definite sotto React.AriaAttributes, e sono già incluse nelle props dei vari tag html (infatti in vs code hai autocomplete :)).
Cosa significa all'atto pratico?
Non devi aggiungere esplicitamente una prop `ariaHidden`, ne un giorno un `ariaQualcosa`. Tienile sotto ...rest quando vai a fare lo spread. Un giorno sarebbe bello passare su tutte le props con ts per definirle a modino ed evitare problemi (metto una prop a caso anche se so che sto lavorando con un componente che renderizza solo dei particolari tag html)
Puoi passarle poi dove servono con {...rest} omettendolo dove non serve, tipo magari XTwitterSVG qui sotto

Reply:
Concordo con Martina. Mi sembra molto meglio ora.